integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -6579,6 +6584,14 @@ msgstr "Weitere Seiten-Infos"
 msgid "This organization currently has no maintained pages."
 msgstr "Von dieser Organisation werden zur Zeit keine Seiten verwaltet."
 
+#: cms/templates/organizations/organization_form.html
+msgid "View location"
+msgstr "Weitere Ort-Infos"
```

Comment:
```suggestion
msgstr "Ort ansehen"
```
Hm, this translation is not really fitting :thinking:

Reply:
It was inherited from the already existing "View page".
The suggestion is applied in both.

Reply:
Strange! But thank you very much! :)